cpp/tests/test_epi_data_io.cpp

```diff
@@ -301,6 +331,34 @@ TEST(TestEpiDataIo, read_confirmed_cases_data)
     ASSERT_EQ(case_data[2].state_id, boost::none);
 }
 
+TEST(TestEpiDataIo, read_confirmed_cases_noage_data)
+{
+    auto rki_data_noage =
+        mio::read_confirmed_cases_noage(mio::path_join(TEST_DATA_DIR, "cases_all_germany.json")).value();
+
+    ASSERT_EQ(rki_data_noage.size(), 15);
+
+    ASSERT_EQ(rki_data_noage[0].date, mio::Date(2020, 05, 24));
+    ASSERT_EQ(rki_data_noage[0].num_confirmed, 1.2);
+    ASSERT_EQ(rki_data_noage[0].num_deaths, 0.5);
+    ASSERT_EQ(rki_data_noage[0].num_recovered, 0);
+
+    ASSERT_EQ(rki_data_noage[8].date, mio::Date(2020, 06, 01));
+    ASSERT_EQ(rki_data_noage[8].num_confirmed, 44);
+    ASSERT_EQ(rki_data_noage[8].num_deaths, 8);
+    ASSERT_EQ(rki_data_noage[8].num_recovered, 3);
+
+    ASSERT_EQ(rki_data_noage[9].date, mio::Date(2020, 06, 02));
+    ASSERT_EQ(rki_data_noage[9].num_confirmed, 44.5);
+    ASSERT_EQ(rki_data_noage[9].num_deaths, 8.999);
+    ASSERT_EQ(rki_data_noage[9].num_recovered, 6);
+
+    ASSERT_EQ(rki_data_noage[14].date, mio::Date(2020, 06, 07));
+    ASSERT_EQ(rki_data_noage[14].num_confirmed, 120.6);
+    ASSERT_EQ(rki_data_noage[14].num_deaths, 30);
+    ASSERT_EQ(rki_data_noage[14].num_recovered, 6);
+}
```

Comment:
Again: where is the value in testing multiple days?

Reply:
Also: In the other tests, we also tested more than one day. I also tried to target integer values as well as double values for the entries.